content/modules/r/pages/rep.adoc

```diff
@@ -0,0 +1,105 @@
+= `rep`
+
+`rep` is short for replicate. `rep` accepts some object, `x`, and up to three additional arguments: `times`, `length.out`, and `each`. `times` is the number of non-negative times to repeat the whole object `x`. `length.out` specifies the end length you want the result to be. `rep` will repeat the values in `x` as many times as it takes to reach the provided `length.out`. `each` repeats each element in `x` the number of times specified by `each`.
```

Comment:
`rep` (short for replicate) accepts an object `x` and up to 3 additional arguments, `time`, `length.out`, and `each`. 

`times`: number of (non-negative) times to repeat the whole object, `x`
`length.out`: the length you want your result to be
`each`: number of times to repeat each element in `x`